core/src/main/java/org/infinispan/util/logging/Log.java

```diff
@@ -270,10 +268,10 @@
    @Message(value = "Unexpected error in AsyncStoreCoordinator thread. AsyncCacheWriter is dead!", id = 55)
    void unexpectedErrorInAsyncStoreCoordinator(@Cause Throwable t);
 
-   @SuppressWarnings("deprecation")
-   @LogMessage(level = ERROR)
-   @Message(value = "Exception reported changing cache active status", id = 58)
-   void errorChangingSingletonStoreStatus(@Cause SingletonCacheWriter.PushStateException e);
+//   @SuppressWarnings("deprecation")
```

Comment:
Should we just remove this? Or are we reserving log message ids?

Reply:
I wasn't sure what to do either, I saw some other messages nearby were only commented so I did the same.

Reply:
Okay, I am fine doing as is, maybe @tristantarrant can provide input as to if we can reuse the id in the future or not.